specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json

```diff
@@ -97,13 +97,14 @@
     }
   },
   "definitions": {
-    "FabricLocation": {
-      "description": "Place holder for fabric location.",
+   "FabricLocation": {
```

Comment:
Is this model not being used before?

Reply:
Yes. Previously the FabricLocation had no properties. In recent changes we have added properties to the resource.